che #17338 Fixing the misleading procedure step for enabling metrics.  Adding a note that metrics enabled by default when Eclipse Che is installed via the OperatorHub (#1366)
codenvy/che-docs: src/main/pages/che-7/administration-guide/proc_enabling-and-exposing-che-metrics.adoc

.Procedure

. Set the `CHE_METRICS_ENABLED=true` environment variable, which will expose the `8087` port as a service on the che-master host.

> When {prod} is installed via the OperatorHub, the environment variable will be set automatically if the default `CheCluster` CR is used: 

Reviewer: ```suggestion
When {prod} is installed from the OperatorHub, the environment variable is set automatically if the default `CheCluster` CR is used: 
```
